src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <nav class="stars stars--0">
+      <section class="stars__star"></section>
```

Comment:
[nav](https://www.geeksforgeeks.org/html-nav-tag/) - used for declaring the navigational section
[section](https://www.w3schools.com/tags/tag_section.asp) - used for defines a section in a document

I think in this case we should  `ul` and `li`